Tests/DefaultXpringClientTest.swift

```diff
@@ -265,6 +266,88 @@ final class DefaultXpringClientTest: XCTestCase {
     XCTAssertThrowsError(try xpringClient.getTransactionStatus(for: .testTransactionHash))
   }
 
+  // MARK: - Account Existence
```

Comment:
What does the `MARK` here mean? Is this part of Given, When, Then testing?

Reply:
This may be one for Keefer, resident Swift expert, but I noticed that the MARK comments show up as large text in the file side view in Xcode (the view that shows as much of the file as possible at once with super micro text), making it easy to navigate the test file.

Reply:
Yup, these are just stylistic additions which help Xcode display things in a pretty fashion.  They make these pretty lines and headers in the navigation. 

<img width="467" alt="Screen Shot 2020-03-03 at 3 47 24 PM" src="https://user-images.githubusercontent.com/18271723/75792742-4a21d500-5d66-11ea-9047-909602550d82.png">